src/ImageDetails.jsx

```diff
@@ -9,37 +9,31 @@ const _ = cockpit.gettext;
 
 const ImageDetails = ({ containers, image, showAll }) => {
     return (
-        <DescriptionList isHorizontal className='image-details' columnModifier={{
-            default: '2Col'
-        }}>
+        <DescriptionList className='image-details' isAutoFit>
+            {image.Command !== "" &&
             <DescriptionListGroup>
-                <DescriptionListTerm>{_("ID")}</DescriptionListTerm>
-                <DescriptionListDescription title={image.Id}>{utils.truncate_id(image.Id)}</DescriptionListDescription>
-            </DescriptionListGroup>
-            <DescriptionListGroup>
-                <DescriptionListTerm>{_("Tags")}</DescriptionListTerm>
-                <DescriptionListDescription>{image.RepoTags ? image.RepoTags.join(" ") : ""}</DescriptionListDescription>
+                <DescriptionListTerm>{_("Default command")}</DescriptionListTerm>
```

Comment:
I know you copied it from the design document - but I feel like Default is not needed here. I would just keep it Command. What do you think @marusak ?

Reply:
https://docs.docker.com/engine/reference/builder/#cmd

:man_shrugging: It is kind of default command, but also not super important to spell out "default"...